orcid-api-web/tutorial/search.md

```diff
@@ -119,6 +119,34 @@ Example response
 
  Like all API calls, search queries are restricted by the [API limits](https://github.com/ORCID/ORCID-Source/tree/master/orcid-api-web#api-limits).
 
+ ## Using ORCID search with CSV
+
+ | Item              |Parameter               |
+|-------------------|--------------------------|
+| URL 				| https://pub.sandbox.orcid.org/v3.0/csv-search/[version]/search/?=[query]&fl[csv fields]
+| Header		|'Accept: text/csv' |
+|Method| GET|
+| Endpoint 	|csv-search |
+|Allowed CSV Fields      | orcid, email, given-names, family-name,given-and-family-names, current-institution-affiliation-name,past-institution-affiliation-name, credit-name, other-names|
+
+You can search ORCID with the API and return your search as a CSV as an alternative to JSON and XML. This can be achieved by changing the header, the endpoint and then adding the fields you wish to have returned in your CSV. The search part of the query remains the same.
```

Comment:
then specify the fields you want in the response using the fl (field list) parameter

Reply:
Thanks updated